docs/site/Calling-other-APIs-and-Web-Services.md

```diff
@@ -9,12 +9,121 @@ summary:
 ---
 
 Your API implementation often needs to interact with REST APIs, SOAP Web
-Services or other forms of APIs.
+Services, gRPC microservices, or other forms of APIs.
 
-How to:
+To facilitate calling other APIs or web services, we introduce `@loopback/service-proxy`
+module to provide a common set of interfaces for interacting with backend services.
 
-- Set up a Service
-- Use services with controllers
-- Reuse models between services and controllers
+## Installation
 
-{% include content/tbd.html %}
+```
+$ npm install --save @loopback/service-proxy
```

Comment:
FWIW, `--save` is no longer needed with the npm version shipped in Node.js 8.x LTS, at least AFAIK. If you decide to change this line, then please check other places in this pull request that are using `--save` argument.